articles/migrations/guides/liveid-api-deprecation.md

```diff
@@ -0,0 +1,135 @@
+
+***Raw Profile***
+
+The raw user profile that can be obtained calling the [Get User By Id Endpoint](https://auth0.com/docs/api/management/v2#!/Users/get_users_by_id) has several differences when using Live SDK or Microsoft Graph.
+
+The JSON below shows the fields that are in the Live SDK profile but are not present, or are different, in the Microsoft Graph one:
+
+```js
+{
+    "locale": "en_ZA",
+    "work": [],
+    "emails": [
+        "john.doe@windowslive.com"
+    ],
+    "addresses": {
+        "personal": {
+            "street": null,
+            "street_2": null,
+            "city": null,
+            "state": null,
+            "postal_code": null,
+            "region": null
+        },
+        "business": {
+            "street": null,
+            "street_2": null,
+            "city": null,
+            "state": null,
+            "postal_code": null,
+            "region": null
+        }
+    },
+    "phones": {
+        "personal": null,
+        "business": null,
+        "mobile": null
+    },
+    "nickname": "john.doe@windowslive.com",
+   ]
+}
+```
+
+This other one shows the fields that are in the Microsoft Graph profile but are not present, or are different, in the Live SDK one one:
+
+```js
+{
+    "strategy_version": 2,
+    "displayName": "John Doe",
+    "userPrincipalName": "john.doe@windowslive.com",
+    "businessPhones": [],
+    "nickname": "john.doe",
+}
+```
+
+Key differences are:
+
+- `strategy_version` has the value `1` when the user last logged-in with Live Connect, or `2` when they last logged in with Azure AD.
+- `nickname` has a different format. 
+- `locale`, `work`, `emails` array, `addresses`, `phones` are present in Live SDK but not in Microsoft Graph.
+- `displayName`, `userPrincipalName`, `businessPhones` array are present in Microsoft Graph but not in Live SDK.
+
+If a user that has previously logged in with Live Connect logs in with AzureAD, the profiles will be merged. The new profile will have both the content of the Live ID profile plus the fields from the Microsoft Graph profile.
+
+The `strategy_version` will be set to '2' for users that last logged in with Azure AD, to '1' for users that logged in with Live Connect recently, and will not be present for users that did not login recently. You can use this field to better interpret the profile content.
+
+Note that the `user_id` field will be the same regardless of the API used to connect with Microsoft.
+ 
+**Permissions**
+
+Auth0 lets you select which permissions you want to ask from the Microsoft Graph APIs. The ones that Live SDK and Microsoft Graph support might provide similar functionality, but the data returned by them and their format may be completely different. See [Migrating from Live SDK - Permissions](https://docs.microsoft.com/en-us/onedrive/developer/rest-api/concepts/migrating-from-live-sdk?view=odsp-graph-online#permissions) to understand what changes are required in your code.
+
```

Comment:
Is this table trying to match items on the same row? There seems to be a complete mismatch. 
Also, will we try to translate from one set of permissions to the other? Or are the permissions cleared when switching strategies? MIght be worth clarifying that.

Reply:
Permissions are cleared, since the APIs are different. Agreed about the mismatch, the mapping is already documented here: https://docs.microsoft.com/en-us/onedrive/developer/rest-api/concepts/migrating-from-live-sdk?view=odsp-graph-online#permissions so maybe we don't need to have a table here to avoid duplication or things not in sync.